tools/api-markdown-documenter/src/test/snapshots/html/simple-suite-test/default-config/index.html

```diff
@@ -1,43 +1,28 @@
-<!DOCTYPE html>
+<!doctype html>
```

Comment:
For reviewers: most of the formatting changes boil down to:

- Casing of `doctype`
- Not line wrapping lowest level tag blocks (mostly paragraphs, headings, etc.)